interface/main/holidays/Holidays_Controller.php

```diff
@@ -0,0 +1,109 @@
+<?php
```

Comment:
Just be persistent and keep trying to go to the link. For some reason sourceforge has been throwing out a 500 error about 25% of the time over the last couple hours.